features/step_definitions/login_steps.rb

```diff
@@ -23,4 +60,44 @@
     When I am on the landing page
     When I click "Login with Google"
   }
+end
+
+And(/^I am a logged-out "([^"]*)"$/) do |user_type|
```

Comment:
Necessary step for logged-out users to be recognized as admin / staff / students